### PR TITLE
build: use clang-8

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

Description:
Use Clang-8 in CI.

Risk Level: Low
Testing: CI
Docs Changes:
Release Notes:

### CODE REVIEW

**Comment by lizan** [comment]
The code change is generated by `clang-format-8`.

cc @htuch for cc_wrapper review.

File: bazel/cc_wrapper.py [hunk 1]
```diff
@@ -71,6 +71,10 @@ def main():
   else:
     argv += sys.argv[1:]
 
+  # Enforce LLD if LLD is in argv, by default the latest -fuse-ld will be used
+  if "-fuse-ld=lld" in argv:
```
Comment (silentdai) [inline bazel/cc_wrapper.py]: in -> **not** in?

Comment (lizan) [reply bazel/cc_wrapper.py]: No this is intended to be `in`. Bazel will add `-fuse-ld=gold` in some cases, gcc/clang will take the last -fuse-ld argument, so whenever we specified lld once, add it to last.

**Comment by lizan** [comment]
@silentdai follow https://apt.llvm.org/ to install clang-8 in ubuntu 18.04

File: bazel/cc_wrapper.py (hunk 1 above)
Comment (silentdai) [reply bazel/cc_wrapper.py]: OMG. Thank you for the explanation!

Comment (htuch) [reply bazel/cc_wrapper.py]: Please do update the comments with this :)

**Comment by htuch** [comment]
@lizan do you reckon we can merge this soon? I think this will fix the oss-fuzz build that has been down the past couples of days; https://oss-fuzz-build-logs.storage.googleapis.com/log-6eaaf9b1-cadc-4a49-bfa1-cbe577b6782b.txt. It looks like it's using `gold` rather than `lld`.

**Comment by lizan** [comment]
@htuch this should be good to go, can you force merge if only clang_tidy is failing? From https://circleci.com/gh/envoyproxy/envoy/195947 I see it fails on win32/macos header not found which I will investigate why `bazel query` included them in https://github.com/envoyproxy/envoy/issues/6554

**Pull request merged**